plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBExecUtils.java

```diff
@@ -569,8 +569,8 @@ public static DBSEntityAssociation getAssociationByAttribute(DBDAttributeBinding
     public static boolean equalAttributes(DBCAttributeMetaData attr1, DBCAttributeMetaData attr2) {
         return
             attr1 != null && attr2 != null &&
-            SQLUtils.compareAliases(attr1.getLabel(), attr2.getLabel()) &&
-            SQLUtils.compareAliases(attr1.getName(), attr2.getName()) &&
+            SQLUtils.compareAliases(CommonUtils.notEmpty(attr1.getLabel()), CommonUtils.notEmpty(attr2.getLabel())) &&
+            SQLUtils.compareAliases(CommonUtils.notEmpty(attr1.getName()), CommonUtils.notEmpty(attr2.getName())) &&
```

Comment:
Maybe it would be better to make `SQLUtils.compareAliases` accept nullable values?

Reply:
I thought about it, but I'm not sure.